### ISSUE TITLE
Can't download Youtube Music titles

### ISSUE THREAD

**Issue by kirgizmustafa17** [yt-dlp/yt-dlp#5546]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting a bug unrelated to a specific site
- [X] I've verified that I'm running yt-dlp version **2022.11.11** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Provide a description that is worded well enough to be understood

I am tryig to download a Top 100 Chart on YouTube Music but there is issue.

Now first music title is named **Belki** and it's url is **https://music.youtube.com/watch?v=AsnyAOaQ8so** but yt-dlp is trying to download **https://www.youtube.com/watch?v=Qfbgn8shRUA**

As you can see the original song has **AsnyAOaQ8so** ID
![8so](https://user-images.githubusercontent.com/55243398/201865750-75759094-5d0d-4e03-a167-1534e91b31f2.jpg)

And the video version of that song has **Qfbgn8shRUA** ID
![RUA](https://user-images.githubusercontent.com/55243398/201865738-1d3c7f8c-2f00-4cea-90cd-f74e684a8e5b.jpg)

`yt-dlp -vU --print url https://music.youtube.com/playlist?list=PL4fGSI1pDJn5tdVDtIAZArERm_vv4uFCR --flat-playlist --playlist-items 1` this command will give me **Qfbgn8shRUA** ID but song has **AsnyAOaQ8so** ID.

I can download the songs as format 141 one by one but if I to download playlist or chart yt-dlp will download the 251 format for bestaudio.

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['-vU', '--print', 'url', 'https://music.youtube.com/playlist?list=PL4fGSI1pDJn5tdVDtIAZArERm_vv4uFCR', '--flat-playlist', '--playlist-items', '1']
[debug] Portable config "C:\cli\yt-dlp.conf": ['--path', 'D:/Muhtar/Downloads/yt-dlp']
[debug] Encodings: locale cp1254, fs utf-8, pref cp1254, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2022.11.11 [8b64402] (win_exe)
[debug] Python 3.8.10 (CPython AMD64 64bit) - Windows-10-10.0.19044-SP0 (OpenSSL 1.1.1k  25 Mar 2021)
[debug] exe versions: ffmpeg 2022-04-07-git-607ecc27ed-full_build-www.gyan.dev (setts), ffprobe 2022-04-07-git-607ecc27ed-full_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.15.0, brotli-1.0.9, certifi-2022.09.24, mutagen-1.46.0, sqlite3-2.6.0, websockets-10.4
[debug] Proxy map: {}
[debug] Loaded 1723 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Latest version: 2022.11.11, Current version: 2022.11.11
yt-dlp is up to date (2022.11.11)
[debug] [youtube:tab] Extracting URL: https://music.youtube.com/playlist?list=PL4fGSI1pDJn5tdVDtIAZArERm_vv4uFCR
[youtube:tab] PL4fGSI1pDJn5tdVDtIAZArERm_vv4uFCR: Downloading webpage
[youtube:tab] PL4fGSI1pDJn5tdVDtIAZArERm_vv4uFCR: Redownloading playlist API JSON with unavailable videos
[download] Downloading playlist: Top 100 Songs Turkey
[youtube:tab] Playlist Top 100 Songs Turkey: Downloading 1 videos of 100
[debug] The information of all playlist entries will be held in memory
[download] Downloading video 1 of 1
https://music.youtube.com/watch?v=Qfbgn8shRUA
[download] Finished downloading playlist: Top 100 Songs Turkey
```

**Comment by dirkf** [comment]
Continued from discussion in #5502 where the issue was associated with Chart-type playlists, like `PL4fGSI1pDJn5tdVDtIAZArERm_vv4uFCR`, in particular.

**Comment by atmazzerexe** [comment]
Added these comments for more contexts.
Also @kirgizmustafa17 I'd suggest you to change the title to "Unable to download YouTube Music Premium Audio (141) when download Chart-type Playlist" for more clarity on the issue.
Comment 1:
https://github.com/yt-dlp/yt-dlp/issues/5502#issuecomment-1314249298 
Comment 2:
https://github.com/yt-dlp/yt-dlp/issues/5502#issuecomment-1315705881

**Comment by pukkandan** [comment]
Also related: https://github.com/yt-dlp/yt-dlp/issues/622

**Comment by atmazzerexe** [comment]
> Does premium users get the non-premium version of the playlist with https://youtube.com/playlist?list=PL4fGSI1pDJn6puJdseH2Rt9sMvt9E2M4i ?
> 
> If so, this is similar to #622

Yeah, YouTube Premium users [will get the exact same playlist as non-premium ones](https://user-images.githubusercontent.com/14852623/202000756-e423c87d-0cb7-4316-bb55-90b81b19fa2e.jpg). Also true, I can't download premium audio if I use `youtube.com` as my link even if it's using the same video ID as the `music.youtube.com` one, which aligns well with how the browser version behave anyway.

<details>
<summary>YouTube.com audio format on YouTube Music ID links</summary>

![image](https://user-images.githubusercontent.com/14852623/202000355-125d37ec-b009-4b26-9704-3174991d168b.png)

</details>

**Comment by kirgizmustafa17** [comment]
I think I found the problem. If you open the that playlist in private tab, all urls in playlist will shown as Video URL. But if you login to your account you will see the Music URL of music titles.

**Comment by Chartman123** [comment]
As a workaround you can select the titles in the "charts playlist" and add them to an own private playlist. That way I was able to download the songs in 141 quality.

**Comment by Raprodent** [comment]
Just to be clear: AAC 256kpbs ( format ID tag 141 ) is totally gone on all videos, including music videos from YouTube Music Premium? This is insane. It existed before?

**Comment by gamer191** [comment]
@Raprodent no, this issue is specifically talking about "chart" playlists ([types of playlists](https://github.com/yt-dlp/yt-dlp/issues/5502#issuecomment-1314249298))
